.github/workflows/rust.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: build
-          args: --workspace --lib --no-default-features --target thumbv7em-none-eabi --exclude wasmi_cli
+          args: --workspace --lib --no-default-features --target thumbv7em-none-eabi --exclude wasmi_cli --exclude wasmi_wasi
       - name: Build (wasm32)
```

Comment:
This branch also needs to exclude `wasmi_wasi`, sorry! So just add another `--exclude wasmi_wasi` there.

Reply:
@Robbepop you mean the `wasm32` build?

Reply:
yes exactly

Reply:
it is what is causing CI to fail atm